exec/java-exec/src/test/java/org/apache/drill/exec/record/vector/TestValueVector.java

```diff
@@ -560,17 +561,20 @@ public void testBitVector() {
       m.set(0, 1);
       m.set(1, 0);
       m.set(1, 0);
+      m.setValueCount(2);
       assertEquals(1, accessor.get(0));
       assertEquals(0, accessor.get(1));
 
       // test toggling the values
       m.set(0, 0);
       m.set(1, 1);
+      m.setValueCount(2);
       assertEquals(0, accessor.get(0));
       assertEquals(1, accessor.get(1));
 
       // Ensure unallocated space returns 0
-      assertEquals(0, accessor.get(3));
+      // Not a valid check since bounds checking is enabled
+     //assertEquals(0, accessor.get(3));
```

Comment:
since you have removed the bounds check let's uncomment this assert. Also comment says unallocated space shouldn't it be unassigned index instead ?